### PR TITLE
#### What are the relevant tickets?

### PR DESCRIPTION
**Pull request by blarghmatey**
None

#### What's this PR do?
Adds the UJSON renderer to the Django Rest Framework

#### How should this be manually tested?
Validate that the API is still able to send and receive JSON data.

#### Any background context you want to provide?
The UJSON renderer is up to 2x or 3x faster than the default one that ships with Python.

### CODE REVIEW

**Comment by noisecapella** [comment]
Speaking of parsing JSON I saw this recently, about the corner cases of JSON parsing that are treated differently by different parsers: http://seriot.ch/parsing_json.html

Though I feel like if our tests are passing we should generally be good for that

File: micromasters/settings.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
 from django.core.exceptions import ImproperlyConfigured
 
 
-VERSION = "0.20.0"
+VERSION = "0.18.0"
```
Comment (noisecapella) [inline micromasters/settings.py]: Can you revert this?

**Comment by noisecapella** [comment]
Looks like everything still works, merge if tests pass :+1:

**Comment by blarghmatey** [comment]
This is actually showing a negative performance impact and increased error rate during load tests. Abandoning this PR.
